yt/yt/server/lib/misc/estimate_size_helpers-inl.h

```diff
@@ -8,12 +8,37 @@ namespace NYT {
 
 ////////////////////////////////////////////////////////////////////////////////
 
+template <typename I>
+    requires std::integral<I>
+size_t EstimateSize(const I& /*value*/)
+{
+    return EstimatedValueSize;
+}
```

Comment:
Why do we estimate integers as 16 bytes instead of 8? Furthermore, why are floating points 32 bytes?

Reply:
Idk, this code is quite ancient: https://github.com/ytsaurus/ytsaurus/commit/a2a80a90bcec869fffd63b7d9a89bbc26e21d929. 
I just went with the spirit of things, since `TGuid` is also estimated as 32 bytes even though it is only 16 bytes in reality. 

Frankly, I do not like the way this is implemented at all. The best way to estimate the size of an unversioned row is to call `GetDataWeight` on the actual row. Is there any reason we cannot materialize the unversioned rows earlier and throw out all of this code? The rows will be materialized later on anyway. I guess it might be a bit cumbersome to ensure that rows we drop are deallocated at the right moment, but it should be doable.
In any case, that is obviously a more substantial refactoring that I do not want to pursue from a non code-owning point of view.

Reply:
So in this case I am more for keeping coherence with the existing logic.